templates/server.xml.epp

```diff
@@ -87,7 +87,7 @@
                     keystoreType="<%= $jira::tomcat_keystore_type %>"
 <%   if $jira::proxy { -%>
 <%     jira::sort_hash($jira::proxy).each |$key, $value| { -%>
-                    <%= key %> = '<%= $value %>'
+                    <%= $key %> = '<%= $value %>'
```

Comment:
that really looks like a bug that should have been catched by the unit tests. @dacron could you add a test for this that verifies the xml content?

Reply:
I've added a test.